lib/galaxy/schema/__init__.py

```diff
@@ -1,13 +1,44 @@
-import typing
+from typing import (
+    Dict,
+    List,
+    Optional,
+)
 
 from pydantic import BaseModel
 
 
 class BootstrapAdminUser(BaseModel):
     id = 0
-    email: typing.Optional[str] = None
-    preferences: typing.Dict[str, str] = {}
+    email: Optional[str] = None
+    preferences: Dict[str, str] = {}
     bootstrap_admin_user = True
 
     def all_roles(*args) -> list:
         return []
+
+
+class Tour(BaseModel):
```

Comment:
Should we put all the API BaseModels for all the routes under `schema/__init__.py` or should they be in their own file like `schema/tours.py`? I am still struggling with Python packages/modules/__init__ best practices 😅

Reply:
They should :)

Reply:
Done! `tours/schema.py`.